docuilib/src/css/custom.css

```
@@ -30,6 +31,147 @@ html[data-theme='dark'] .docusaurus-highlight-code-line {
   background-color: rgba(0, 0, 0, 0.3);
 }
 
+/** Docusaurus Navbar */
+
+.navbar {
+  height: 86px;
+  padding-top: 16px;
+  padding-bottom: 16px;
+}
+.navbar__logo {
+  height: 53px;
+  margin-right: 76px;
+  margin-left: 36px;
+}
```

Add this to align to bottom, it's a little too centered
```suggestion
}
.navbar__items {
  align-items: flex-end;
}
```